Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "GPL-3.0"
 name = "self_encryption"
 readme = "README.md"
 repository = "https://github.com/maidsafe/self_encryption"
-version = "0.30.0"
+version = "0.30.1"
```

Comment:
Just a quick question. If this is a breaking change, do we want to bump to `0.31.0`?

Reply:
Ah it's not breaking this one.